sessionctx/variable/sysvar.go

```diff
@@ -386,7 +393,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal, "log_syslog_include_pid", ""},
 	{ScopeSession, "last_insert_id", ""},
 	{ScopeNone, "innodb_ft_cache_size", "8000000"},
-	{ScopeNone, "log_bin", "OFF"},
+	{ScopeNone, LogBin, boolToStatusStr(config.GetGlobalConfig().Binlog.Enable)},
```

Comment:
set here seems not work, `defaultSysVars` is used in  `init` function, this function will run when this path be imported, and the Binlog.Enable may be not set at this time.